frontend/README.md

```diff
@@ -49,65 +51,109 @@ WARNING: Do not store sensitive data in the environment variables.
 
 ## Deployment
 
-1.  Ensure your `awscli` is configured with the
-    [required credentials and profiles](../docs/awscli.md).
-    Set the appropriate `AWS_PROFILE`.
+1. Ensure your `awscli` is configured with the
+   [required credentials and profiles](../docs/awscli.md).
+   Set the appropriate `AWS_PROFILE`.
+
+   ```shell
+   export AWS_PROFILE=single-cell-dev
+   ```
+
+1. **Specify deployment.**
+
+   Set the `DEPLOYMENT_STAGE` environment variable to a valid deployed environment: `dev`, `staging`
+
+   ```shell
+   export DEPLOYMENT_STAGE=dev
+   ```
+
+1. **Deploy.**
+
+   Files are deployed to a publicly accessible bucket. Do not include sensitive data in the deployed files.
+
+   ```shell
+   make deploy
+   ```
+
+## Smoke Test
+
+### Update Tests
+
+#### Gatsby Dev Build
+
+1. Start FE server: `npm run develop`
+
+2. Run tests: `npm run e2e`
+
+#### Gatsby Prod Build
+
```

Comment:
What is the difference between this test and the above?

Reply:
Added description under `### If you are actively adding/updating tests` section to distinguish the two!

> ... The upside of running prod build is that the experience is closer to what real users get, the downside is that it takes longer to build the app and run the tests.